internal/sys/fs.go

```diff
@@ -56,209 +56,6 @@ type FileEntry struct {
 	File fsapi.File
 }
 
-const direntBufSize = 16
-
-// Readdir is the status of a prior fs.ReadDirFile call.
-type Readdir struct {
-	// cursor is the current position in the buffer.
-	cursor uint64
```

Comment:
the cursor is no longer needed in the new version, because the `window` is a `sliceReaddir` which keeps its own internal cursor.